docs/how_to/use_optimizationlevel.rst

```diff
@@ -0,0 +1,296 @@
+#####################################
+Setting transpiler optimization level
+#####################################
+
+This guide shows you how to use the :attr:`~qiskit.transpile.optimization_level` 
+parameter with :meth:`~qiskit.compiler.transpile`.
+:attr:`~qiskit.transpile.optimization_level` helps you to optimize your quantum circuit.
+This parameter takes an integer which can be a value between 0 and 3,
+where the higher the number the more optimized the result.
+You can find more information about this parameter
+`here <https://qiskit.org/documentation/tutorials/circuits_advanced/04_transpiler_passes_and_passmanager.html#Preset-Pass-Managers>`__.
+
+Initialize the quantum circuit
+==============================
+
+The effect of setting the :attr:`~qiskit.transpile.optimization_level` will differ depending on the backend you are using.
+For this example you will exploring the `CSWAP <https://qiskit.org/documentation/stubs/qiskit.circuit.QuantumCircuit.cswap.html>`__ gate,
+a three qubit gate which will be transpiled into one and two qubit gates.
+
+.. testcode::
+
+    from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister 
+    from qiskit.compiler import transpile
+    from qiskit.providers.fake_provider import FakeQuito #fake backend ibmq_quito
+
+    backend = FakeQuito() # For this example the quito fake backend is used
+
+    qr = QuantumRegister(3) #Initialize the Quantum Register 
+    qc = QuantumCircuit(qr) # Initialize the quantum circuit 
+    
+    qc.cswap(qr[0],qr[1],qr[2]) #add the cswap gate to the quantum circuit
+    
+    print(qc) #print the quantum circuit
+
+.. testoutput::
+
+    q0_0: ─■─
+           │ 
+    q0_1: ─X─
+           │ 
+    q0_2: ─X─
+             
+Using backend’s information
+===========================
+
+To apply :meth:`~qiskit.providers.fake_provider.FakeQuito` to your quantum circuit, you should adhere to its specific configurations 
+when you utilize the :meth:`~qiskit.transpile` method. This process entails breaking down your circuit into :attr:`~qiskit.transpile.basis_gates`, 
+and considering the physical connections specified in the :attr:`~qiskit.transpile.coupling_map`` for two qubit gates. 
+Given the presence of noise in the backend, it is crucial to optimize your circuit by adjusting the :attr:`~qiskit.transpile.optimization_level` parameter. 
+This will help minimize the number of circuit operations and enhance the overall performance.
+
+If you set the optimization_level to 0, the coupling map for your backend will be **[[0,1],[1,0],[1,2],[2,1]]**, 
+indicating the physical qubits in the backend. In this configuration, the quantum circuit is transformed into a combination of one and two-qubit gates,
+represented by the **['id', 'rz', 'sx', 'x', 'cx', 'reset']**.
```

Comment:
It might be nice to show how to access this information, e.g. via `backend.configuration()`

Reply:
I think the same, only using backend.configuration() doesn't show the information.
I'm considering that this tutorial is only focus in the parameters that are affected when we use optimization_level, something like this


    print("Basis gates of your Backend: ",backend.configuration().basis_gates)
    print("Coupling map of your Backend: ",backend.configuration().coupling_map)

    Basis gates of your Backend:  ['id', 'rz', 'sx', 'x', 'cx', 'reset']
    Coupling map of your Backend:  [[0, 1], [1, 0], [1, 2], [1, 3], [2, 1], [3, 1], [3, 4], [4, 3]]